.eslintrc.js

```diff
@@ -18,5 +18,6 @@ module.exports = {
       js: 'always',
     }],
     'no-continue': 'off',
+    camelcase: 'off',
```

Comment:
Maybe use `disable-next-line` because we would want to enforce this normally right?
You can also get around this by **not** destructuring the items at `dashboard.js:297`